akka-docs/src/test/java/jdocs/future/FutureDocTest.java

```diff
@@ -11,6 +11,8 @@
 import scala.concurrent.Await;
 import scala.concurrent.Promise;
 import akka.util.Timeout;
+
+
```

Comment:
Two new lines inserted for all `//#import*` as I merged import and actual code snippets for Java like below:

```
- @@snip [FutureDocTest.java]($code$/java/jdocs/future/FutureDocTest.java) { #imports1 }
- @@snip [FutureDocTest.java]($code$/java/jdocs/future/FutureDocTest.java) { #diy-execution-context } 

+ Java
+ :  @@snip [FutureDocTest.java]($code$/java/jdocs/future/FutureDocTest.java) { #imports1 #diy-execution-context } 
```